tests/integration_tests/test_openai_embedding_function.py

```
@@ -0,0 +1,217 @@
+# Skip this test by default - it requires external API access and API keys
+@pytest.mark.skipif(not os.environ.get("OPENAI_API_KEY"), reason="OPENAI_API_KEY environment variable is not set")
+class TestOpenAIEmbeddingFunction:
+    """Test OpenAIEmbeddingFunction - skipped by default, requires manual execution"""
+
+    def test_litellm_env(self):
+        """Test if litellm is installed."""
+        try:
+            import litellm
+        except ImportError:
+            print("litellm package is not installed")
+            assert False
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)